pyproject.toml

```diff
@@ -81,13 +81,20 @@ commands =
         docs/source docs/build
 
 [testenv:release]
-basepython=python3.8
 deps = pep517
 whitelist_externals =
     cp
     rm
 commands =
     rm -rf {toxinidir}/dist
-    cp --no-target-directory --recursive {distdir} {toxinidir}/dist; # copy the wheel built by tox-wheel
+    cp -r {distdir} {toxinidir}/dist; # copy the wheel built by tox-wheel
     {envpython} -m pep517.build --source --out-dir={toxinidir}/dist {toxinidir}
+
+[testenv:publishcov]
+passenv = CI GITHUB_* TRAVIS TRAVIS_* CODECOV_*
```

Comment:
See https://github.com/codecov/codecov-python#using-tox

Maye use `passenv = TOXENV CI TRAVIS TRAVIS_* CODECOV_*`

But I am +1 on just using codecov outside of tox to puslish the coverage.

For me, for coverage publishing, tox just adds an extra layer which is not needed.

Reply:
codecov doesn't collect `GITHUB_` metadata :(